browser/scripts/application.js

```diff
@@ -1529,47 +1529,56 @@ Application.prototype.openSaveACopyDialog = function() {
 	return dfd.promise
 }
 
-Application.prototype.growl = function(title, type, duration, person) {
-	var letter=title.charAt(0);
-	var image=''
-	type= type || 'info';
-	if (!$('symbol#icon-'+type).length) {
-		type='info'
-	}
-	if (person) {
-		var image='<div style="background-color: '+person.color+';" class="image-crop"><span>'+letter+'</span></div>';
-	} 
-	
-	/** TODO: when users will have pics - use this:
-	if (person.userpic) {
-		image = '<div style="background-image: url('+person.userpic+');" class="image-crop"></div>';
+Application.prototype.growl = function(message, type, duration, user) {
+	user = user || null
+	type = type || 'info'
+	duration = duration || 2000
+
+	if (typeof user !== 'object') {
+		user = {
```

Comment:
it's generally a bad practice to change function input values, because the called doesn't expect them to change. call the function parameter 'username' and make a new var user?